docs/guidance/aws-eks-gpu-cluster.md

```diff
@@ -0,0 +1,58 @@
+# Prepare Amazon EKS cluster with GPU for KubeRay examples
+
+## Step 1: Create a Kubernetes cluster on Amazon EKS
+
+Follow the first two steps in this [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#)
+to: (1) Create your Amazon EKS cluster (2) Configure your computer to communicate with your cluster.
+
+## Step 2: Create node groups for the Amazon EKS cluster
+
+You can follow "Step 3: Create nodes" in this [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#) to create node groups. The following section provides more detailed information.
+
+### Create a CPU node group
+
+Typically, we do not run GPU workloads on the Ray head. Create a CPU node group for all Pods except Ray GPU 
+workers, such as the KubeRay operator, Ray head, and CoreDNS Pods.
+
+### Create a GPU node group
+
+Create a GPU node group for Ray GPU workers. Add a Kubernetes taint to prevent scheduling CPU Pods on this GPU node group. For KubeRay examples, we add the following taint to the GPU nodes: `Key: ray.io/node-type, Value: worker, Effect: NoSchedule`.
+
+Next, **please follow Step 4 to install the NVIDIA device plugin.**
+
+> Warning: GPU nodes are extremely expensive. Please remember to delete the cluster if you no longer need it.
+
+## Step 3: Verify the node groups
+
+If you encounter permission issues with `eksctl`, you can navigate to your AWS account's webpage and copy the
```

Comment:
Could you make this a note?

Reply:
https://github.com/ray-project/kuberay/pull/1181/commits/4d018b2fb5561bffb45c83eb0bcf6b7c9d1db9f5